docs/events/key.md

```diff
@@ -0,0 +1,11 @@
+# Key
+
+The `Key` event is sent to a widget when the user presses key on the keyboard.
+
+- [x] Bubbles
+
+## Attributes
+
+| attribute | type | purpose                  |
+| --------- | ---- | ------------------------ |
+| `key`     | str  | The key that was pressed |
```

Comment:
The “purpose” column in the attributes tables for events looks more like a “description” column, no? Exhibit A.